src/axom/inlet/Inlet.hpp

```diff
@@ -249,13 +251,18 @@ class Inlet : public SchemaCreator
    */
   bool get(const std::string& name, std::string& value);
 
+  void registerDocWriter(std::shared_ptr<DocWriter> writer);
+
   // TODO add update value functions
 private:
   axom::sidre::View* baseGet(const std::string& name);
 
   std::shared_ptr<Reader> m_reader;
   axom::sidre::Group* m_sidreRootGroup = nullptr;
   std::shared_ptr<Table> m_globalTable;
+
+  std::shared_ptr<DocWriter> docWriter;
```

Comment:
Please prefix all member variables with "m_" so docWriter should be m_docWriter.